src/app/execution-context.js

```diff
@@ -0,0 +1,81 @@
+var $ = require('jquery');
+var Web3 = require('web3');
+
+var injectedProvider;
+
+if (typeof web3 !== 'undefined') {
+  injectedProvider = web3.currentProvider;
+  web3 = new Web3(injectedProvider);
+} else {
+  web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
+}
+
+function ExecutionContext (compiler) {
```

Comment:
I do not think this is the "execution context". I think this should stay in renderer.js (or something which handles the UI).  By execution context I've meant the core part of universal-dapp.js, which handles:
- account/balance mangement
- transaction processing
- and hides the differences between web3 and VM